iothub/service/src/ServiceClient.cs

```diff
@@ -42,11 +43,11 @@ internal ServiceClient()
         }
 
         /// <summary>
-        /// Create ServiceClient from the specified connection string
+        /// Create an instance of ServiceClient from the specified IoT Hub connection string.
         /// </summary>
-        /// <param name="connectionString">Connection string for the IoT Hub</param>
+        /// <param name="connectionString">Connection string for the IoT Hub.</param>
         /// <param name="options">The options that allow configuration of the service client instance during initialization.</param>
```

Comment:
I know this already exists but adding <see cref="ServiceClientOptions"> for options might be helpful.